src/towncrier/test/test_project.py

```diff
@@ -46,25 +39,61 @@ def test_tuple(self):
         version = get_version(temp, "mytestproja")
         self.assertEqual(version, "1.3.12")
 
-    @skipIf(metadata_version is None, "Needs importlib.metadata.")
     def test_incremental(self):
         """
-        An incremental Version __version__  is picked up.
+        An incremental-like Version __version__  is picked up.
```

Comment:
what does `incremental-like` version means ?

I guess that this means a `__version__` object/instance that has the same public API/interface as incremental.
That is, it doesn't have to be `incremental` it only needs to behave like it.

Reply:
Correct. The code doesn't check the instance of the __version__, just that it quacks like an incremental version.